docs/cpu_templates/schema.json

```diff
@@ -5,6 +5,32 @@
     "description": "Custom CPU template language description",
     "type": "object",
     "properties": {
+        "kvm_capabilities": {
+            "type": "array",
+            "items": {
+                "description": "Additional KVM capabilities can be added or existing (built in) capabilities can be removed from the firecracker checks. To add KVM capability to the checklist specify decimal number of the corresponding KVM capability. To remove a KVM capability from the checklist specify decimal number of the corresponding KVM capability with '!' mark in the front. Works on both x86_64 and aarch64.",
+                "type": "string",
+                "examples": ["171", "!172"]
+            }
+        },
+        "vcpu_features": {
+            "type": "array",
+            "items": {
+                "description": "vCPU features to enable during vCPU initialization. Only for aarch64.",
+                "type": "object",
+                "properties": {
+                  "index": {
+                        "description": "Index into kvm_vcpu_init::features array. Current linux kernel (6.4.10) only allows value 0.",
+                        "type": "integer"
+                  },
+                  "bitmap": {
+                        "description": "Bitmap for modifying the 32 bit field in kvm_vcpu_init::features. Must be in the format `0b[01x]{32}`. Corresponding bits will be cleared (`0`), set (`1`) or left intact (`x`). (`_`) can be used as a separator.",
```

Comment:
Looks like the example doesn't follow the `0b[01x]{32}` pattern as it has fewer digits.

Reply:
There is no pint to specify all 32 bits. Kernel only allows usage of first 7 (as of right now).

Reply:
then we need to relax the regex to accept fewer digits.

Reply:
I think we had an discussion about it and it appeared too complex...

Reply:
I don't think it does actually. Looks like `0b[01x]{1,32}` would be enough.

Reply:
Ok done